src/main/java/com/ipiecoles/java/java350/model/Employe.java

```diff
@@ -120,8 +139,21 @@ else if (this.performance == null || Entreprise.PERFORMANCE_BASE.equals(this.per
         return Math.round(prime * this.tempsPartiel * 100)/100.0;
     }
 
-    //Augmenter salaire
-    //public void augmenterSalaire(double pourcentage){}
+    /**
+     * Méthode qui augmente le salaire par un certain pourcentage. Le pourcentage autorisé est un double
+     * entre 0.0 et 1.0. Le pourcentage maximal autorisé est de 1.0 (100%) afin d'empêcher une augmentation
+     * trop importante.
+     * Pour rappel: un double ne peut pas être "null".
+     *
+     * @param pourcentage
+     * @throws IllegalArgumentException
+     */
+    public void augmenterSalaire(double pourcentage) throws IllegalArgumentException {
```

Comment:
Pour info, IllegalArgumentException est une Run Time Exception donc il ne faut pas ajouter le throws
```suggestion
    public void augmenterSalaire(double pourcentage) throws IllegalArgumentException {
```

Reply:
Tout à fait !